apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -0,0 +1,34 @@
+import {BalanceQuantity} from '@yoroi/types/lib/balance/token'
+import BigNumber from 'bignumber.js'
```

Comment:
this definitely goes in the swap module, after we extract Quantity/Amount helpers into @yoroi/wallet